tools/provision

```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/env bash
+
+echo "Setup: Setting up Virtual Env"
+
+BASEDIR=`dirname $0`/..
+
+if [ ! -d "$BASEDIR/env" ]; then
+    virtualenv $BASEDIR/env --no-site-packages
```

Comment:
I used a local virtualenv here since this is only for the developers to develop in an isolated environment so that they do not clutter their system and in general for a clean environment. It is not meant for use by external scripts.

The users would install the packages using `pip install zulip-api` and `pip install `zulip-bots`. Here, users includes any part in the Zulip server that needs to run a bot or its scripts.